src/Tools/Auditor/DafnyAuditor.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DAFNY_ROOT Condition="'$(DAFNY_ROOT)' == ''">..\..\..\..\</DAFNY_ROOT>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(DAFNY_ROOT)\Source\Dafny\DafnyPipeline.csproj" />
```

Comment:
Once this is compilable with mainline Dafny, could we make it depend on the `DafnyPipeline` library?

Reply:
Yes, I'd love that.

Reply:
Let's create an issue for this one

Reply:
#16